### DISCUSSION TITLE
Unable to download Github Desktop

### DISCUSSION THREAD

**Discussion by fafaraway** [community/community#42120] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can't download GithubDesktop, when I go to `desktop.github.com` it automatically redirects to `github.com`

`https://central.github.com/deployments/desktop/desktop/latest/win32`
When I used this link to try to download it directly, I was met with a 404 page

![image](https://user-images.githubusercontent.com/156806/208434712-650f1bc9-1daf-41ef-b4fd-7bca4ed9c6d7.png)

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, I see that you wrote in a couple of days ago and, when I try on my end, navigating to https://desktop.github.com/ does take me to the GitHub Desktop home page. Can you try again and let us know if you're still seeing this issue on your end? 

In the meantime, I've gone ahead and moved this to the correct category for you. Good luck!
